uint64 inc
uint8 ultrasonicF
uint8 ultrasonicB